### ISSUE TITLE
Access Issues with Data Tracker and HR apps

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#6831]
> What application are you using?

Data Tracker

> Describe the problem.

No aspects of Knack work for me other than Data Tracker.  When I go to my bookmark for Data Tracker, I always have an error message about my email/password (see 1st screenshot).  When I click- Log In, however, it allows me to go to Data Tracker.  If I try to go to the knack homepage or HR page (for the TLC form for example), it does not work- I click Log In and it takes me back to the same exact Log In page (no error message or anything- see 2nd screenshot).  I've tried changing my City/computer password, my knack password and neither has helped.  I still get the same response for both pages.

> Website Address

https://atd.knack.com/hr#new-telework-request/

Internet Browser: Chrome

> Requested By
Erin B.

[Attachment](https://api.knack.com/v1/applications/595d00ebd315cc4cb98daff4/download/asset/61043faf7993ad001e9bb19d/capture2.png) (51.78kb)

Request ID: DTS21-102435

**Comment by dianamartin** [comment]
Amenity reached out to Erin
>Thanks for your support ticket and sorry you're having issues with Data Tracker. What browser are you using? 

E replied
>Chrome

A replied
>Hmmm… and you’re able to log into all the other Office 365 stuff in the browser with that password? “Anything but Internet Explorer” is the main rule, but If you have time to check Firefox that could be helpful to isolate the issue. (I think most of us use Chrome; just may be an extension, cache thing, etc.)

E replied
>Ok, let me check Firefox. It does the same thing in Firefox

**Comment by dianamartin** [comment]
I reached out to Erin, did a phone/screen share. 

She showed me using the bookmark she had it had a knack error message saying she couldn't log in. I resent her the Data Tracker URL and she deleted the old bookmark and saved the new link. She tried it and it worked perfectly.

She was also trying to access the TLC request form and I knew already it was bc she was most likely not in the system. I check and she wasn't. I added her in the system and she was able to sign in and load the form.
